diff --git a/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js b/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
index 1bb80f7..0363ca6 100644
--- a/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
+++ b/node_modules/@jest-runner/electron/build/rpc/JestWorkerRPC.js
@@ -46,7 +46,7 @@ const _runInBrowserWindow = testData => {
     const workerID = (0, _utils.makeUniqWorkerId)();
     const win = new _electron.BrowserWindow({
       show: false,
-      webPreferences: {nodeIntegration: true, contextIsolation: false}
+      webPreferences: {nodeIntegration: true, contextIsolation: false, enableRemoteModule: true}
     });
 
     win.loadURL(`file://${require.resolve('../index.html')}`);
